well_architected_framework_validation/update-srg-id.sh

```diff
@@ -1,9 +1,9 @@
-sed -e "s/\$RELEASE_PRODUCT/$RELEASE_PRODUCT/g" -e "s/skip: true/skip: false/g" ./srg/cost/_config_template.yaml > ./srg/cost/_config_generated.yaml
-sed -e "s/\$RELEASE_PRODUCT/$RELEASE_PRODUCT/g" -e "s/skip: true/skip: false/g" ./srg/opr-excel/_config_template.yaml > ./srg/opr-excel/_config_generated.yaml
-sed -e "s/\$RELEASE_PRODUCT/$RELEASE_PRODUCT/g" -e "s/skip: true/skip: false/g" ./srg/perf/_config_template.yaml > ./srg/perf/_config_generated.yaml
-sed -e "s/\$RELEASE_PRODUCT/$RELEASE_PRODUCT/g" -e "s/skip: true/skip: false/g" ./srg/reliability/_config_template.yaml > ./srg/reliability/_config_generated.yaml
-sed -e "s/\$RELEASE_PRODUCT/$RELEASE_PRODUCT/g" -e "s/skip: true/skip: false/g" ./srg/security/_config_template.yaml > ./srg/security/_config_generated.yaml
-sed -e "s/\$RELEASE_PRODUCT/$RELEASE_PRODUCT/g" -e "s/skip: true/skip: false/g" ./srg/sustainability/_config_template.yaml > ./srg/sustainability/_config_generated.yaml
-sed -e "s/\$RELEASE_PRODUCT/$RELEASE_PRODUCT/g" -e "s/skip: true/skip: false/g" ./workflow/_config_wf_actions_template.yaml > ./workflow/_config_wf_actions_generated.yaml
-sed -e "s/\$RELEASE_PRODUCT/$RELEASE_PRODUCT/g" -e "s/skip: true/skip: false/g" ./workflow/_config_wf_template.yaml > ./workflow/_config_wf_generated.yaml
+sed -e "s/\$RELEASE_PRODUCT/$RELEASE_PRODUCT/g" -e "s/skip: true/skip: false/g" ./srg/cost/_config_template.yamlx > ./srg/cost/_config_generated.yaml
```

Comment:
is there a reason for the change to `yamlx` file-endings? To me this would imply extended YAML syntax, which some libraries have created, but Monaco does not support. I'd keep these as .yaml unless there's a good reason not to

Reply:
what I am doing here is to update some params in this template file (.yamlx) outside of monaco with a script and changing the type to "yaml" to be recognized by monaco. In previous version, it was also with the same "yaml" extension and there was a duplicate config issue. It could be anything than yamlx for this template config file